awx/ui_next/src/screens/Template/TemplateList/TemplateList.jsx

```diff
@@ -58,7 +54,7 @@ function TemplateList({ i18n }) {
         jtActions: results[1].data.actions,
         wfjtActions: results[2].data.actions,
       };
-    }, [location, projectId]),
+    }, [location]),
```

Comment:
instead of location, could we use the useParams hook?

Reply:
was totally wrong...useParams gets the params inside the route definition, like ":id".  the search (stuff after the "?") is not what it means by params.